example_client/faces_analysis_pipeline_client.py

```diff
@@ -32,7 +32,8 @@
 parser.add_argument('--face_images_save_path', required=False, default='', help='If specified, face images will be saved to disk.')
 parser.add_argument('--image_width', required=False, default=600, help='Original image width. default: 600')
 parser.add_argument('--image_height', required=False, default=400, help='Original image height. default: 400')
-parser.add_argument('--image_layout', required=False, default='CHW', choices=['CHW', 'HWC'], help='Original image layout. default: CHW')
+parser.add_argument('--input_image_layout', required=False, default='CHW', choices=['CHW', 'HWC'], help='Original image layout. default: CHW')
```

Comment:
help sentence is a bit missleading. what is the original? maybe model input layout would be clearer?

Reply:
This client is removed on main branch and will be removed from develop anyway.

Reply:
Pipeline input image layout

Reply:
CHW -> NHWC